ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -35,6 +46,91 @@ def _set_base_span_tags(
             else:
                 span.set_tag_str(API_KEY, api_key)
 
+    def llmobs_set_tags(
+        self,
+        resp: Any,
+        span: Span,
+        args: List[Any],
+        kwargs: Dict[str, Any],
+        err: Optional[Any] = None,
+    ) -> None:
+        """Extract prompt/response tags from a completion and set them as temporary "_ml_obs.*" tags."""
+        if not self.llmobs_enabled:
+            return
+
+        parameters = {
+            "temperature": float(kwargs.get("temperature", 1.0)),
+            "max_tokens": float(kwargs.get("max_tokens", 0)),
+        }
+        messages = get_argument_value([], kwargs, 0, "messages")
+        system_prompt = get_argument_value([], kwargs, 0, "system")
```

Comment:
Based on anthropic's create() function signature it doesn't make any sense to try pulling these from `args`. Let's just pull them using `kwargs.get(...)` instead

Reply:
word, ill make that change